sql/sql_list.h

```diff
@@ -52,6 +52,14 @@ class SQL_I_List :public Sql_alloc
     next= elements ? tmp.next : &first;
   }
 
+  SQL_I_List &operator=(const SQL_I_List &tmp)
```

Comment:
These operators / copy constructors doesn't really work together with other parts of this class... not that we can do anything about this, as looks like they are used already :(

Reply:
My thinking is that before these changes assignment operator was already there (a default generated one with bitwise copying). So, having it here defined explicitly shouldn't do any harm. Moreover, the initialization of the `next` member is handled better in this case.